### PR TITLE
chore: remove experimental label following bazel 6.0.0 release

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -70,14 +70,10 @@ const Home: NextPage<HomePageProps> = ({ searchIndex }) => {
       <main className="m-4 l:m-0">
         <div className="max-w-4xl w-4xl mx-auto mt-8 flex flex-col items-center">
           <h1 className="text-bzl-green font-bold text-6xl">
-            Bazel Central Registry
+            Bazel Module Registry
```
Comment (Wyverald) [inline pages/index.tsx:73]: revert this?

Comment (alexeagle) [reply pages/index.tsx:73]: That was intentional as part of rewording the H2 content below it to be less informal. The "central registry" is the other repository. But sure, I'm not that opinionated about it.

**Comment by hobofan** [comment]
If I'm not mistaken, as of 6.0 bzlmod still needs to be explicitly enabled via a flag right?

So I think it would make sense to keep the infobox with a notice linking to the user guide that tells newcomers how to enable it (though in a less warning color).

**Comment by alexeagle** [comment]
That spot in the UI seems kinda awkward for technical documentation, like what would you propose to put there? A grey box that says "Bazel 6.0: Put `common --enable_bzlmod` in your .bazelrc. Earlier Bazel: see docs" ?

**Comment by hobofan** [comment]
My suggestion would be: "As of version 6.0, Bazel comes with a new external dependency system. To get started please take a look at the <a>user guide</a>."

Given that the link to the homepage is the one that's being circulated the most, I think that would strike a good balance of informing/managing expectations and pointing people that want to try it out for the first time into the right direction.

**Comment by jsharpe** [comment]
This also removes experimental status from the BCR which isn't the same thing as bzlmod being made generally available because it's certainly possible to use bzlmod without a central registry?

Let's be sure that this is actually what we intend..

**Comment by Wyverald** [comment]
James has a good point -- let's hold this off until BCR official launch has been cleared. Due to the artifact mirroring drama, it was not strictly coupled with Bazel 6.0 anymore -- though it shouldn't be too far off (hopefully this week).

**Review by hobofan — APPROVED** [review]

**Pull request merged**